tests/src/common/WskCli.java

```diff
@@ -976,7 +982,14 @@ public RunResult cli(boolean verbose, int expectedExitCode, String... params) th
     public RunResult cli(boolean verbose, int expectedExitCode, File workingDir, String... params) throws IllegalArgumentException, IOException {
         String[] cmd = verbose ? Util.concat(baseCmd, "--verbose") : baseCmd;
         String[] args = Util.concat(cmd, "-i", "--apihost", WhiskProperties.getEdgeHost());
-        RunResult rr = TestUtils.runCmd(DONTCARE_EXIT, workingDir, TestUtils.logger, this.env, Util.concat(args, params));
+        String[] finalArgs = Util.concat(args, params);
+        if (this.verbose == true) {
+            System.out.println("CLI RUNNING WITH ARGS:\n" + Arrays.toString(finalArgs));
```

Comment:
this you can get by setting the `logger` verbosity to `info` vs the default `warning` level.